corputils/views.py

```diff
@@ -58,7 +87,7 @@ def corp_member_view(request, corpid = None):
             corpid = membercorplist[0][0]
 
     corp = EveCorporationInfo.objects.get(corporation_id=corpid)
-    Player = namedtuple("Player", ["main", "maincorp", "maincorpid", "altlist", "apilist"])
+    #Player = namedtuple("Player", ["main", "user", "maincorp", "maincorpid", "altlist", "apilist", "n_fats"])
```

Comment:
And the reason that I had to create a separate class. The named tuples does not allow for reasignment of variables.